common/tests/fixtures/ethereum/sign_typed_data.json

```diff
@@ -4,6 +4,28 @@
         "passphrase": ""
     },
     "tests": [
+        {
+            "name": "domain_only_message",
+            "comment": "Bare minimum EIP-712 message",
+            "parameters": {
+                "path": "m/44'/60'/0'/0/0",
+                "metamask_v4_compat": true,
+                "data": {
+                    "types": {
+                        "EIP712Domain": []
```

Comment:
Theoretically not necessary, but I would also welcome a test-case with non-empty domain-data

Reply:
Added in 82e1e9693a57ab77a819abf2f6e180925e47c620, although I haven't tested whether it works with Trezor One yet, since I still haven't had the time to figure out how to run tests for it. I'll let CI do it.